Jenkinsfile

```diff
@@ -381,10 +408,44 @@ pipeline {
                 ]
             )
             script {
-                println '====== Archive artifacts'
-                println "Artifact 1: ${_ARTIFACT_COVERAGE}"
-                archiveArtifacts artifacts: "${_ARTIFACT_COVERAGE}", allowEmptyArchive: true, onlyIfSuccessful: false
+                println '====== Archive jacoco reports artifacts'
+                archiveArtifacts artifacts: "${'**/jacoco-aggregate/**/*.*'}", allowEmptyArchive: true, onlyIfSuccessful: false
             }
         }
     }
 }
+
+/**
+ * Assembly all needed items to put inside extraBuildParams
+ *
+ * @param Boolean skip_doc, if set to true documentation build will be skipped
+ *
+ * @return extraBuildParams as a string ready for mvn cmd
+ */
+private String extraBuildParams_assembly(Boolean skip_doc) {
+    String extraBuildParams
+
+    println 'Processing extraBuildParams'
+    println 'Manage the EXTRA_BUILD_PARAMS'
+    final List<String> buildParamsAsArray = []
+
+    println 'Manage user params'
+    if ( params.EXTRA_BUILD_PARAMS )
+        buildParamsAsArray.add( params.EXTRA_BUILD_PARAMS as String )
+
+    println 'Manage the skip_doc option'
+    if (skip_doc) {
+        buildParamsAsArray.add('--define skipAntora=true')
+        buildParamsAsArray.add('--define antora.skip=true')
+        buildParamsAsArray.add('--define component.front.build.skip=true')
+        buildParamsAsArray.add('--define talend.documentation.pdf.skip=true')
+        buildParamsAsArray.add('--projects !org.talend.sdk.component:documentation')
```

Comment:
what's the difference with the statement below? Is it really recognized?

Reply:
Actually, I am not sure if this one is used **skipAntora** we should look it together
**antora.skip** doesn't work alone, and did request me to add **component.front.build.skip** and **talend.documentation.pdf.skip** .

Would it be equivalent to do only **org.talend.sdk.component:documentation** ?
I will do a check.

Reply:
Finally, I improve it, but it requests to remove reporting as well, It is OK @undx